Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml

```diff
@@ -472,75 +354,66 @@ script:
       description: The replyTo recipients of the email.
       type: String
   - arguments:
-    - default: false
-      description: The ID of the message.
-      isArray: false
+    - description: The ID of the message.
       name: message_id
       required: true
-      secret: false
-    - default: false
-      description: The comment of the replied message.
-      isArray: false
+    - description: The comment of the replied message.
       name: comment
       required: true
-      secret: false
-    - default: false
-      description: A comma-separated list of email addresses for the 'to' field.
+    - description: A comma-separated list of email addresses for the 'to' field.
       isArray: true
       name: to
       required: true
-      secret: false
-    - default: false
-      description: A comma-separated list of War Room entry IDs that contain files,
+    - description: A comma-separated list of War Room entry IDs that contain files,
         which are used to attach files for the email to send. For example, attachIDs=15@8,19@8.
       isArray: true
       name: attach_ids
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of names of attachments to display in the
+    - description: A comma-separated list of names of attachments to display in the
         email to send. Must be the same number of elements as attach_ids.
       isArray: true
       name: attach_names
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of CIDs to embed attachments within the
+    - description: A comma-separated list of CIDs to embed attachments within the
         actual email.
       isArray: true
       name: attach_cids
-      required: false
-      secret: false
-    deprecated: false
     description: The replies to the recipients of a message.
-    execution: false
     name: msgraph-mail-reply-to
   - arguments:
-    - default: false
-      description: The ID of the draft email.
-      isArray: false
+    - description: The ID of the draft email.
       name: draft_id
       required: true
-      secret: false
-    deprecated: false
     description: Sends a draft email using Microsoft Graph.
-    execution: false
     name: msgraph-mail-send-draft
-  - deprecated: false
+  - arguments: []
     description: Tests connectivity of the email.
-    execution: false
     name: msgraph-mail-test
+  - arguments:
+    - description: An OData query. See REDAME for OData usage examples.
```

Comment:
Please add the OData usage examples to the README, as in Microsoft Graph Mail. You can take it from [here](https://github.com/demisto/content/blob/master/Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/README.md#odata-usage) and make the relevant updates (remove the user).

Reply:
> Hi, Thank you for your contribution!
> 
> Please mind the requested changes:
> 
> 1. Remove the unneeded imports that were added.
> 2. Add to the README the command you introduced- this should be done using `demisto-sdk generate-docs` command. You can find more information about it [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md).
> 3. Add unit tests to the commands you added - this can be the same as in Microsoft Graph Mail. Tell me if you need help with it.

I can't generate docs:
```
Error: 'NoneType' object has no attribute 'endswith'
```

Other issues will be addressed soon in the newest pull (just give me 1-2h)

Reply:
Btw i'm running demisto-sdk 1.6.6

Reply:
Have you run it as described [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md)? You should provide the yml file, a commands examples file and the commands name.

Reply:
Yes, I did it